cmd/crank/main.go

```diff
@@ -45,8 +46,17 @@ func (v versionFlag) IsBool() bool { return true }
 
 // BeforeApply indicates that we want to execute the logic before running any
 // commands.
-func (v versionFlag) BeforeApply(app *kong.Kong) error { //nolint:unparam // BeforeApply requires this signature.
-	fmt.Fprintln(app.Stdout, version.New().GetVersionString())
+func (v versionFlag) BeforeApply(app *kong.Kong) error {
+	vxp, err := common.FetchCrossplaneVersion()
+	if err != nil {
+		return err
```

Comment:
Probably worth returning the cli version first so that we show that even with errors.

Reply:
done